src/js/components/uploadDetachedFiles/UploadDetachedFileValidation.jsx

```diff
@@ -258,11 +260,14 @@ export default class UploadDetachedFileValidation extends React.Component {
 		}
 
 		validationBox = <ValidateDataFileContainer type={type} data={this.state.jobResults}/>;
-		if(!this.state.headerErrors && this.state.validationFinished) {
+		if (!this.state.headerErrors && this.state.validationFinished) {
 			validationBox = <ValidateValuesFileContainer type={type} data={this.state.jobResults} setUploadItem={this.uploadFile.bind(this)} updateItem={this.uploadFile.bind(this)} published={this.state.published}/>;
-			validationButton = <button className='pull-right col-xs-3 us-da-button' onClick={this.submitFabs.bind(this)}>Publish</button>;
-			if(this.state.published){
-				validationButton = <button className='pull-right col-xs-3 us-da-disabled-button' disabled>File Already Published</button>;
+
+			if (PermissionsHelper.checkFabsPermissions(this.props.session)) {
```

Comment:
Do we not want to show the button at all for DABS users or do we want to show the button but not let them click it with different text?